calicoctl.py

```diff
@@ -733,19 +729,19 @@ def ip_pool_add(cidr_pool, version):
     Add the the given CIDR range to the IP address allocation pool.
 
     :param cidr_pool: The pool to set in CIDR format, e.g. 192.168.0.0/16
+    :param version: v4 or v6
     :return: None
     """
-    assert version in (4, 6)
```

Comment:
I'd rather we didn't lose asserts when we've taken the time to write them.  Should be able to `assert version in ("v4", "v6")`